docs/pages/technical-specs/expo-update-0.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Expo Update
+sidebar_title: Expo Update
+---
+
+version 0
+
+---
+
+## Introduction
+
+This is the specification for Expo Updates, a protocol for delivering updates to apps running on multiple platforms.
```

Comment:
What should be the name of the protocol. Is it "Expo Update", "Expo Updates" or something else entirely?